c2corg_ui/templates/outing/edit.html

```diff
@@ -555,11 +555,18 @@ <h5 translate ng-show="outing.associations.users.length == 0">You can look up fo
             </div>
 
             ## COMMENT
-            <div class=" full form-group">
+            <div class="full form-group">
               <label translate>personal comments</label>
               <textarea app-markdown-editor class="form-control description" placeholder="{{'write your comments' | translate}}" ng-model="outing.locales[0]['description']"></textarea>
             </div>
 
+            ## COMMENTS MANAGEMENT
+            <div class="form-group half">
+              <label translate>disable_comments</label>
+              <input type="checkbox" ng-model="outing.disable_comments">
+              <span ng-click="editCtrl.pushToArray(outing, 'disable_comments', !outing.disable_comments, $event)" translate>If checked, comments cannot be posted or read.</span>
```

Comment:
I guess you checked, but just to be sure. Is it really `!outing.disable_comments` (negated)?